crates/network/src/any/builder.rs

```diff
@@ -96,18 +96,36 @@ impl TransactionBuilder<AnyNetwork> for WithOtherFields<TransactionRequest> {
         self.deref_mut().set_gas_limit(gas_limit);
     }
 
-    fn build_unsigned(self) -> BuilderResult<<AnyNetwork as Network>::UnsignedTx> {
-        build_unsigned::<AnyNetwork>(self.inner)
+    /// Get the EIP-2930 access list for the transaction.
+    fn access_list(&self) -> Option<&AccessList> {
+        self.deref().access_list()
+    }
+
+    /// Sets the EIP-2930 access list.
+    fn set_access_list(&mut self, access_list: AccessList) {
+        self.deref_mut().set_access_list(access_list)
     }
 
-    fn get_blob_sidecar(&self) -> Option<&BlobTransactionSidecar> {
-        self.deref().get_blob_sidecar()
+    fn blob_sidecar(&self) -> Option<&BlobTransactionSidecar> {
```

Comment:
dropping the get_ prefix is good

Reply:
matches the style of the rest of all the other methods